### PR TITLE
Charts enhancements

### PR DESCRIPTION
**Pull request by daneryl**
PieChart support to show labels instead of tooltip via showLabels="true" prop
PieChart accepts innerRadius and outerRadius props
PieChart automatically filters '0' values
PieChart/BarChart support for max categories to show via maxCategories prop
BarChart vertical layout support via layout='vertical' prop

PR checklist:
- [ ] Update READ.me ?
- [ ] Update API documentation ?

QA checklist:
- [ ] Smoke test the functionality described in the issue
- [ ] Test for side effects
- [ ] UI responsiveness
- [ ] Cross browser testing
- [ ] Code review

### CODE REVIEW

File: app/react/Markdown/components/PieChart.js [hunk 1]
```diff
@@ -60,17 +60,25 @@ export const PieChartComponent = (props) => {
 
 PieChartComponent.defaultProps = {
   context: 'System',
+  innerRadius: '0',
+  outerRadius: '105',
   classname: '',
   colors: '#ffcc00,#ffd633,#ffe066,#ffeb99,#fff5cc',
   data: null,
+  showLabel: 'false',
```
Comment (RafaPolit) [inline app/react/Markdown/components/PieChart.js]: Mmmm... I think I agree with having this default as false, but then that would 'break' the existing Pies.  Any thoughts about that? Would it merit to revert the default?

File: app/react/Markdown/components/BarChart.js [hunk 2]
```diff
@@ -51,21 +65,25 @@ export const BarChartComponent = (props) => {
 
 BarChartComponent.defaultProps = {
   context: 'System',
+  layout: 'horizontal',
+  maxCategories: '0',
```
Comment (RafaPolit) [inline app/react/Markdown/components/BarChart.js]: This will certainly break every pie and chart out there.  Should we default to something 'useful'?

**Comment by RafaPolit** [comment]
- Append the 'others' at the end of the array
- If maxCategories has more than the values, it works, but appends a 0 value after the array (a quick fix is to move the zero sanitizing after the appending)

**Comment by daneryl** [comment]
@RafaPolit proposed fixes implemented

**Pull request merged**